build_image/api/redoc.py

```diff
@@ -35,6 +36,9 @@ async def app_lifespan(app):
 
 app = FastAPI(openapi_url=None, lifespan=app_lifespan)
 
+# Serve files from the 'api/assets' directory under the '/static' URL path
```

Comment:
I think it's fairly obvious what this line does, but maybe less so why you are doing this. Maybe leave a short comment about this instead